sniffr/user_routes/user_routes.py

```diff
@@ -15,6 +14,37 @@ def delete_user(user_id):
 
         return {'message': f"Successfully deleted user"}, 410
 
+    else:
+        response = {"message": "User Not Found"}
+        return response, 404
+
+@user_bp.route("/user/edit", methods=["POST"])
+@token_required
+def edit_user(current_user):
+    """Edits a user."""
+    content = request.json
+
+    user_id = current_user.user_id
+    queried_user = db.session.query(User).filter(User.user_id==user_id).first()
+
+    if queried_user:
+        queried_user.email = content["email"]
+        queried_user.password = content["password"]
+        queried_user.username = content["username"]
```

Comment:
Do we need to hash the password in this function before committing it to the database?

Reply:
Good catch, Dan! We definitely need that. I was thinking maybe we can add this into another story where we research and install any package necessary for this change. @allie-rae thoughts?

Reply:
The password field is automatically hashed and already working. Check out the models.py file to view the hashing function.